onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.h

```diff
@@ -25,6 +25,8 @@ static const std::string kForceSequentialEngineBuild= "ORT_TENSORRT_FORCE_SEQUEN
 static const std::string kEngineCachePath = "ORT_TENSORRT_ENGINE_CACHE_PATH";
 static const std::string kDecryptionEnable = "ORT_TENSORRT_ENGINE_DECRYPTION_ENABLE";
 static const std::string kDecryptionLibPath = "ORT_TENSORRT_ENGINE_DECRYPTION_LIB_PATH";
+static const std::string kDLAEnable = "ORT_TENSORRT_DLA_ENABLE";
+static const std::string kDLACore = "ORT_TENSORRT_DLA_CORE";
```

Comment:
shouldn't we add these to provider options and not add any more environment variables?

Reply:
Last time we talked about to probably change provider options for CUDA and TRT. Is there any conclusion? And also in the future do we want to keep both env variables and provider options?

Reply:
Had a sync up with @jywu-msft offline. Agree to remove has_trt_options and include all env variables in provider options. Probably it's better to make those changes in a separate PR.